javaslang/src/main/java/javaslang/collection/BitMappedTrie.java

```diff
@@ -83,22 +81,19 @@ private static int treeSize(int branchCount, int depthShift) {
         }
     }
 
-    private BitMappedTrie<T> boxed() { return map(identity()); }
+    BitMappedTrie<T> boxed() { return map(identity()); }
 
```

Comment:
Making it package-private suggests to me that it is called from the outside of BMT. Separation of concerns!?